iphone/Classes/CMHelper.m

```diff
@@ -135,18 +135,18 @@ +(NSDictionary*)dictionaryFromMotionActivity:(CMMotionActivity*)motionActivity
 +(NSDictionary*)dictionaryFromPedometerData:(CMPedometerData*)pedometerData
 {
     NSMutableDictionary *dict = [NSMutableDictionary dictionaryWithDictionary:@{
-        @"startDate": (!pedometerData) ? [NSNull null] : [TiUtils UTCDateForDate:pedometerData.startDate],
-        @"endDate": (!pedometerData) ? [NSNull null] : [TiUtils UTCDateForDate:pedometerData.endDate],
-        @"numberOfSteps": (!pedometerData) ? [NSNull null] : NUMINT(pedometerData.numberOfSteps),
-        @"distance": (!pedometerData) ? [NSNull null] : NUMINT(pedometerData.distance),
-        @"floorsAscended": (!pedometerData) ? [NSNull null] : NUMINT(pedometerData.floorsAscended),
-        @"floorsDescended": (!pedometerData) ? [NSNull null] : NUMINT(pedometerData.floorsDescended)        
+        @"startDate": (!pedometerData.startDate) ? [NSNull null] : [TiUtils UTCDateForDate:pedometerData.startDate],
+        @"endDate": (!pedometerData.endDate) ? [NSNull null] : [TiUtils UTCDateForDate:pedometerData.endDate],
+        @"numberOfSteps": (!pedometerData.numberOfSteps) ? [NSNull null] : pedometerData.numberOfSteps,
+        @"distance": (!pedometerData.distance) ? [NSNull null] : pedometerData.distance,
+        @"floorsAscended": (!pedometerData.floorsAscended) ? [NSNull null] : pedometerData.floorsAscended,
+        @"floorsDescended": (!pedometerData.floorsDescended) ? [NSNull null] : pedometerData.floorsDescended
```

Comment:
Yes, I understand that. I am just wondering how the output value differs with that change. Can you give me an example of values with and without this change? I mean, `NUMINT([NSNumber numberWithInt:5])` and `[NSNumber numberWithInt:5]` will still both return 5, no matter if we downcast it or not.